README.md

```diff
@@ -2,15 +2,22 @@
 <!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
 [![All Contributors](https://img.shields.io/badge/all_contributors-4-orange.svg?style=flat-square)](#contributors-)
 <!-- ALL-CONTRIBUTORS-BADGE:END -->
-A **community-driven**, **open-source**, **Unity3D based** Mount &amp; Blade II : BannerLord module Editor
+
+A **community-driven**, **open-source**, **Unity3D based** Mount &amp; Blade II : BannerLord module Editor.
 
 ## Feature list
 
-* Module Data Visual Editor (**WIP**)
-* Drag/Drop Modify Tool
-* Character/Item/Model Preview
-* Troop Tree Preview
-* Build Pipline (build/debug/deployment)
+* Module Data Visual Editor.   
+  Allow users to import MB2 `xsd` files and edit values. Users can modify values by Draging/Droping.
+
```

Comment:
* Extensionalable Customized Editor component
Allow users to extend themselves editor components for the different data sets.
Eg. Face editor, to edit face  visually instead of giving unreadable data directly